task_01.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""This is a docstring"""
+
+
+import decimal
+
+ABSOLUTE_DIFFERENCE = decimal.Decimal('273.15')
+
+
+def fahrenheit_to_celsius(degrees):
+    """Takes fahrenheit and converts it to celsius.
+
+    This function takes a number (fahrenheit) and converts it into celcius.
+
+    Args:
+        degrees (mixed): A number representing degrees in Fahrenheit
+
+    Returns:
+        celsius (mixed): The celcius dgrees representation of the Fahrenheit
+
+    Example:
+        >>> fahrenheit_to_celsius(98.7)
+        Decimal('37.0555555555555571345394128')
+
+    """
+    degrees = decimal.Decimal(str(degrees))
```

Comment:
str() conversion is unnecessary here because you're already converting it to decimal!